app/src/main/java/com/aspirebudgetingmobile/aspirebudgeting/activities/Splash.java

```diff
@@ -0,0 +1,48 @@
+package com.aspirebudgetingmobile.aspirebudgeting.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.os.Handler;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.content.ContextCompat;
+
+import com.aspirebudgetingmobile.aspirebudgeting.R;
+import com.aspirebudgetingmobile.aspirebudgeting.utils.ObjectFactory;
+
+public class Splash extends AppCompatActivity {
+
+    ObjectFactory objectFactory = ObjectFactory.getInstance();
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_splash);
+
+        try {
+            getWindow().setNavigationBarColor(ContextCompat.getColor(Splash.this, R.color.colorPrimary));
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+
+        if (objectFactory.getUserManager().getLastAccount(Splash.this) != null) {
```

Comment:
Store the instance of UserManager in the class

Reply:
done